metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/SemaphoresPerPluginManager.java

```diff
@@ -38,12 +34,9 @@ public SemaphoresPerPluginManager(int acquireTimeoutSecs, int permissionsPerSema
    *
    * @param executablePluginType the provided executable plugin type
    * @return true if acquisition was successful, false otherwise
-   * @throws InterruptedException if we were interrupted during acquisition
    */
-  public boolean tryAcquireForExecutablePluginType(ExecutablePluginType executablePluginType)
-      throws InterruptedException {
-    return unmodifiableMaxThreadsPerPlugin.get(executablePluginType)
-        .tryAcquire(acquireTimeoutSecs, TimeUnit.SECONDS);
+  public boolean tryAcquireForExecutablePluginType(ExecutablePluginType executablePluginType) {
```

Comment:
Because this no longer throws an InterruptedException, it seems like `WorkflowExecutor.runMetisPluginWithSemaphoreAllocation()` no longer throws one either. It can be removed from that method declaration, and from there possibly it has further effects?

Reply:
That's true, wonder why my IDE didn't say anything..
Removing it, there shouldn't be any side effect so we are good on that perspective